features/checkout/addressing_order/addressing_order_validation.feature

```diff
@@ -0,0 +1,94 @@
+@checkout_addressing
+Feature: Order addressing validation
+    In order to avoid making mistakes when addressing an order
+    As an Administrator
+    I want to be prevented from adding it without specifying required fields
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And I am logged in customer
+
+    @ui
+    Scenario: Address an order without name
+        Given I have product "PHP T-Shirt" in the cart
+        And I proceed with the checkout addressing step
+        Then I do not specify the first name
+        And I do not specify the last name
+        And I specify the street as "100 MAIN ST"
+        And I specify the city as "New York City"
+        And I specify the postcode as "93-554"
+        And I try to proceed with the next step
+        Then I should be notified that the shipping "first name" is required
```

Comment:
Should be something similar to "... that the first name in shipping details is required"